locales-pending/onboarding.ftl

```diff
@@ -19,3 +20,28 @@ onboarding-get-started-how-it-works-dialog-step2-content = We’ll show you what
 onboarding-get-started-how-it-works-dialog-step3-heading = Resolve data breaches
 onboarding-get-started-how-it-works-dialog-step3-content = We’ll give you clear steps on how to resolve each data breach. They’ll require access to your accounts, so you’ll need to resolve them manually even if you upgrade to { -brand-premium }.
 onboarding-get-started-how-it-works-dialog-confirm-label = OK
+
+onboarding-enter-details-title = Enter the details you want to protect
+onboarding-enter-details-text = We’ll use this to find exposures of your personal information, and then guide you step-by-step on how to fix it.
+onboarding-enter-details-why-button-label = Why do we need this info?
+onboarding-enter-details-dialog = There’s a $240 billion industry of data brokers that expose and sell your personal information for a profit. They scrape your data from publicly available sources to create profiles that include your name(s), current and previous addresses, family member names, criminal history, and much much more.
```

Comment:
Oh, didn’t know that we should split up longer texts. Are we doing that somewhere else as well and do we have a convention around how many characters a string should have?

cc @maxxcrawford

Reply:
cc @flodolo (since we're talking .ftl and best practices)

Also, @flodolo, let me know if you want me to add you to [CODEOWNERS](https://github.com/mozilla/blurts-server/blob/main/docs/CODEOWNERS) file for these pending strings. Not sure if that is too much noise for you, or if you prefer reviewing early.